ansible_collections/arista/cvp/docs/_build/cv_container_v3.rst

```diff
@@ -48,7 +48,7 @@ The following options may be specified for this module:
     <td>loose</td>
     <td><ul><li>loose</li><li>strict</li></ul></td>
     <td>
-        <div>Set how configlets are attached/detached on container. If set to strict all configlets not listed in your vars are detached.</div>
+        <div>Set how configlets are attached/detached to containers. If set to strict all configlets not listed in your vars will be detached.</div>
```

Comment:
Should be `to/from` and I guess we should have one sentence

Reply:
This is just to match what's already there in schema documentation. We have to change both later.